packs/equipment/skunk-bomb-greater.json

```diff
@@ -51,7 +51,13 @@
         "reload": {
             "value": "-"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Note",
+                "selector": "{item|_id}-damage",
+                "text": "PF2E.BombNotes.SkunkBomb.Greater.success"
```

Comment:
Consider adding a title.

Reply:
Sure. Would that be instead of the `<strong>Effect</strong>` or additonal?

![image](https://github.com/user-attachments/assets/97e348d2-bd13-4591-87e1-e1f595a7d64e)

Reply:
> Sure. Would that be instead of the `\<strong\>Effect\</strong\>` or additonal?

Hm, looking at the other bombs I see they use `<strong>Effect</strong>` as well. I guess the title is already in the attack name.